rpc/src/rpc_service.rs

```diff
@@ -227,16 +227,37 @@ impl RequestMiddleware for RpcRequestMiddleware {
         trace!("request uri: {}", request.uri());
 
         if let Some(ref snapshot_config) = self.snapshot_config {
-            if request.uri().path() == "/snapshot.tar.bz2" {
+            if request.uri().path() == "/snapshot.tar.bz2"
+                || request.uri().path() == "/incremental-snapshot.tar.bz2"
```

Comment:
Can you turn these into constants? How about:

```rust
const FULL_SNAPSHOT_REQUEST_PATH: &str = "/snapshot.tar.bz2";
const INCREMENTAL_SNAPSHOT_REQUEST_PATH: &str = "/incremental-snapshot.tar.bz2";
```

Reply:
Done